tests/DevApps/ciam/myWebApi/myWebApi.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFrameworks>net7.0</TargetFrameworks>
+    <TargetFrameworks>net8.0</TargetFrameworks>
```

Comment:
unrelated?

Reply:
When the file path for the test apps is calculated, it expects the file to be in the .net 8 target folder. The CIAM app was using net 7 however so I figured the path of least resistance would be to update the app to .NET 8. App appears to be functioning the same.

Reply:
Kind of brittle to have the tfm in the output dir...